plugin/cds-hooks/src/main/java/org/opencds/cqf/ruler/cdshooks/CdsHooksConfig.java

```diff
@@ -39,6 +39,31 @@ public CdsServicesCache cdsServiceInterceptor(IResourceChangeListenerRegistry re
 		return listener;
 	}
 
+	@Bean
+	@Conditional(OnR4Condition.class)
+	public org.opencds.cqf.ruler.cpg.r4.provider.LibraryEvaluationProvider r4LibraryEvaluationProvider() {
+		return new org.opencds.cqf.ruler.cpg.r4.provider.LibraryEvaluationProvider();
```

Comment:
These should be supplied by the CPG plugin. This plugin should depend on that one, and only be enabled if that one is also enabled. Otherwise, enabling both could cause conflicts.